working-groups/best-practices.md

```diff
@@ -2,23 +2,38 @@
+
+## Discussions & decisions
   * Adhere to AMP’s [consensus-based decision-making policy](../GOVERNANCE.md#decision-making-policy).
   * Discussions should be held in a public forum and generally allow for asynchronous communication.  GitHub issues/pull requests and Slack work well for this.
   * Determine and document which types of decisions require proactive approval from the WG.  For most WGs, most work should be able to proceed without requiring an affirmative approval of the WG (e.g. code reviews and smaller feature changes) unless specifically requested by someone in the WG.
   * Document how members and non-members of the WG may request that an item be brought to the WG for a decision and how decisions will be documented.  One option is to add issues requiring WG decisions to a “Pending WG Decision” project.
-* Status updates.
+  
+## Status updates
   * Publish periodic status updates (ideally at least once every two weeks) so the community can keep up to date on the WG’s work without having to follow all issues/Slack conversations/etc.
-  * These updates should include:
-    * what the WG accomplished since the last status update
-    * what they are planning on working on before the next status update
-    * any lessons the WG learned they’d like to share with others in the AMP community
-  * File these updates as issues in the WG’s repo with the label “Type: Status Update.”
+    *  These updates should include:
+      * what the WG accomplished since the last status update
+      * what they are planning on working on before the next status update
+      * any lessons the WG learned they’d like to share with others in the AMP community
+    * File these updates as issues in the WG’s repo with the label “Type: Status Update.”
+  * Each WG will provide a higher-level update to the community/TSC approximately every three months at a Design Review.  See the [Working Group Updates page](https://github.com/ampproject/meta/wiki/Working-Group-Updates) for more information.
+  
+## Working Group Membership
+  * According to [AMP's governance model](https://github.com/ampproject/meta/blob/master/GOVERNANCE.md), the TSC sets the initial membership of each Working Group.  Each Working Group then adjusts its membership as needed using the [consensus-based decision-making policy](../GOVERNANCE.md#decision-making-policy).
+  * Although each Working Group may define its own standards for determining Working Group membership, the following is a set of baseline expectations for Working Group members that most Working Groups follow:
+    * Members of a Working Group are expected to be contributing in a significant way to the work of the Working Group.  Significant code contributions is one way to demonstrate this, but you can also demonstrate this through things like driving an overarching product vision for the WG.
```

Comment:
I think we should be more inclusive of other ways to contribute. My feeling is that the bar here is a bit high. What is an overarching product-vision?

I think that consistent contribution of ideas and feedback should qualify.

Reply:
+1, IMO consistency in contribution (of any form) is key. A person who answers 1 or 2 UI related questions from the community every week make a better WG-UI member than someone who contributed a big feature as the only thing in a year. Any area of contribution like: answering and helping the community with their questions, providing feedback and ideas on discussions, filling issues for bugs and feature requests, contributing code such as bug fix and new features, is all qualifying IMO.

Reply:
How about "Members of a Working Group are expected to consistently contribute to the work of the Working Group.  Contributions can take many forms, including code contributions, feature/design discussions and helping the community."

Reply:
LGTM

Reply:
LG